tests/index.js

```diff
@@ -1,7 +1,7 @@
 const chai = require('chai');  
 const expect = chai.expect;
 
-const Validator = require("../src/");
+var Validator = require("../src");

```

Comment:
Using `const` makes the most sense here. Since we don't want to overwrite the `Validator` variable which contains the library that has been imported. Is there any reason that it has to be changed to `var`.

Reply:
Yes, that makes sense. I'll revert it back.